CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Behavior tests.
```

Comment:
Since this change will not publish a new version. This shouldn't be in the changelog. It's just implementation details.